### PR TITLE
Bugfix: ComponentDetailPanel: boolean attributes with value `false` are removed by `ComponentDetailPanel.sanitizeAttributes`

### LINKED ISSUE #261: ComponentDetailPanel: boolean attributes with value `false` are removed by `ComponentDetailPanel.sanitizeAttributes`
**Describe the bug**

When I set a boolean attribute to false in the "Component details" panel, and I click on <kbd>SAVE</kbd>: I can see that this boolean attribute is not present in the attributes of the component when the `renderFiles` method is called on my plugin's renderer.

This bug is caused by the method `sanitizeAttributes` in the `ComponentDetailPanel`:

https://github.com/ditrit/leto-modelizer/blob/5c2cfb74ed1d339a41c49735d38fa68329400e6b/src/components/drawer/ComponentDetailPanel.vue#L138

**Expected behavior**

Boolean attributes with value `false` should not be removed from the component's attributes.

**Application (please complete the following information):**

- leto version 1.1.0

### PR DESCRIPTION
**Pull request by viennem**
Fix #261

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Fix attribute error management.
 * Fix bug on nested object in attribute edition, see [this issue](https://github.com/ditrit/leto-modelizer/issues/203).
 * Fix bug on model files opening when switching on TextView, see [this issue](https://github.com/ditrit/leto-modelizer/issues/202).
+* Fix bug on boolean attributes with value `false` removed by `ComponentDetailPanel.sanitizeAttributes`, see [this issue](https://github.com/ditrit/leto-modelizer/issues/261).
```
Comment (Zorin95670) [inline changelog.md:55]:
```
* Fix bug on loosing `false` in boolean attributes, see [this issue](https://github.com/ditrit/leto-modelizer/issues/261).
```

Comment (viennem) [reply changelog.md]: fixed

**Review by Zorin95670 — DISMISSED** [review]

File: changelog.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Fix attribute error management.
 * Fix bug on nested object in attribute edition, see [this issue](https://github.com/ditrit/leto-modelizer/issues/203).
 * Fix bug on model files opening when switching on TextView, see [this issue](https://github.com/ditrit/leto-modelizer/issues/202).
+* Fix bug on loosing `false` in boolean attributes, see [this issue](https://github.com/ditrit/leto-modelizer/issues/261).
```
Comment (k-y0u) [inline changelog.md:55]: Fix bug on loosing boolean attributes defined to `false`, see ...

Comment (viennem) [reply changelog.md]: fixed

**Review by Zorin95670 — APPROVED** [review]

**Review by k-y0u — APPROVED** [review]

**Pull request merged**